e2e/test-component/scenarios/embedding-sdk/interactive-question-map.cy.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createQuestion, describeEE, mapPinIcon } from "e2e/support/helpers";
+import {
+  METABASE_INSTANCE_URL,
+  mockAuthProviderAndJwtSignIn,
+  mountInteractiveQuestion,
+  signInAsAdminAndEnableEmbeddingSdk,
+} from "e2e/support/helpers/component-testing-sdk";
+import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+
+const { PEOPLE_ID } = SAMPLE_DATABASE;
+
+describeEE("scenarios > embedding-sdk > interactive-question", () => {
```

Comment:
nit: sync test suite name with file name

```suggestion
describeEE("scenarios > embedding-sdk > interactive-question-map", () => {
```

Reply:
Done, thank you